integration/single-node-slug-deploy/check.go

```diff
@@ -722,10 +723,21 @@ func getConsulManifest(dir string) (string, error) {
 		}
 	}
 
+	interfaces, err := net.Interfaces()
+	if err != nil {
+		return "", util.Errorf("could not list host interfaces to find which to bind consul to: %s", err)
+	}
+	addrs, err := interfaces[0].Addrs()
+	if err != nil {
+		return "", util.Errorf("could not list addresses to find one to bind consul to: %s", err)
+	}
+	ip := strings.Split(addrs[0].String(), "/")[0]
+
 	rubyBin := filepath.Join(dir, "launch")
 	cmds := `#!/usr/bin/env ruby
 
-exec "/usr/bin/consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul"`
+exec "/usr/bin/consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul -bind %%%%"`
```

Comment:
from wikipedia: ```In the context of servers, 0.0.0.0 means "all IPv4 addresses on the local machine". If a host has two IP addresses, 192.168.1.1 and 10.1.2.1, and a server running on the host listens on 0.0.0.0, it will be reachable at both of those IPs.```

And consul has decided that they don't want to bind to multiple IP addresses, so it's a crash in that case. so we choose one arbitrarily here